ui/css/perf.css

```diff
@@ -277,6 +277,21 @@ th {
   color: #333;
 }
 
+.compare-table .subtest-header-visible {
+  color: #333;
+  transition: None;
+}
+.compare-table .subtest-header-visible > th {
+  padding: 5px;
```

Comment:
You can probably use a bootstap utitlity for this instead: https://getbootstrap.com/docs/4.3/utilities/spacing/